lib/parallel_report_portal/cucumber/report.rb

```diff
@@ -223,8 +228,14 @@ def status_to_level(status)
           LOG_LEVELS.fetch(status, LOG_LEVELS[:info])
         end
       end
-      
-    
+
+      def get_report_link(response)
```

Comment:
I’d prefer this to be called `parse_report_link_from_response` or some such. `get_` prefixes on method names are rarely used but this is a useful thing to store.